apiserver/facades/client/storage/storage.go

```diff
@@ -725,6 +730,17 @@ func (a *StorageAPI) Import(args params.BulkImportStorageParams) (params.ImportS
 	return params.ImportStorageResults{Results: results}, nil
 }
 
+// Import imports existing storage into the model.
+// A "CHANGE" block can block this operation.
+func (a *StorageAPIv6) Import(args params.BulkImportStorageParams) (params.ImportStorageResults, error) {
+	for _, storageParam := range args.Storage {
+		if storageParam.Force {
+			return params.ImportStorageResults{}, errors.NotSupportedf("Force import filesystem on this version of Juju")
+		}
+	}
```

Comment:
Each result record has its own error. So the force check is done per record. One might succeed, one might fail etc.

Reply:
Hi, @wallyworld 

Want to know more about your suggestion here.

The check here is just to make sure none *force* argument is pass for api v6. So it fails if **any** force param is detect.

Reply:
Hum, this part removed after the refactoring of adding `ImportStorageParamsV2`. So should be no concern.